tests/helpers.py

```diff
@@ -59,11 +60,11 @@ def parse_responses(response_message_cls, responses: List[proto.Message]) -> byt
     # in order to be an actual JSON.
     json_responses = [
         (
-            response_message_cls.to_json(r).strip('"')
+            response_message_cls.to_json(response).strip('"')
             if issubclass(response_message_cls, proto.Message)
-            else MessageToJson(r).strip('"')
+            else MessageToJson(response).strip('"')
         )
-        for r in responses
+        for response in responses
```

Comment:
```suggestion
        for response in all_responses
```

to not have two variables differ by just one letter....

Reply:
addressed.